batcher/aligned/src/main.rs

```diff
@@ -277,8 +277,16 @@ async fn main() -> Result<(), AlignedError> {
                 SubmitError::IoError(batch_inclusion_data_directory_path.clone(), e)
             })?;
 
-            let max_fee =
-                U256::from_dec_str(&submit_args.max_fee).map_err(|_| SubmitError::InvalidMaxFee)?;
+            if !submit_args.max_fee.ends_with("ether") {
+                error!("Amount should be in the format XX.XXether");
+                return Ok(());
+            }
+
+            let max_fee = submit_args.max_fee.replace("ether", "");
+
+            let max_fee_ether = parse_ether(&max_fee).map_err(|e| {
```

Comment:
This should not be ether. It should be WEI, because submit_multiple receive U256

Reply:
Good catch, I changed the variable names to reflect the unit they are in.